hack/upgrade-test-build-images.sh

```diff
@@ -47,8 +47,14 @@ export REGISTRY_IMAGE_NAME="hco-registry-upgrade"
 export REGISTRY_EXTRA_BUILD_ARGS="--build-arg KUBEVIRT_PROVIDER=$KUBEVIRT_PROVIDER"
 make bundleRegistry
 
+pwd
+make container-clusterserviceversion
```

Comment:
This script only runs on the STDCI. We'll need to have the component operator validation also occur on the OpenShift CI.